lua/lspconfig/julials.lua

```diff
@@ -51,13 +56,31 @@ configs.julials = {
     description = [[
 https://github.com/julia-vscode/julia-vscode
 
-`LanguageServer.jl` can be installed with `julia` and `Pkg`:
+LanguageServer.jl can be installed with `julia` and `Pkg`:
+```sh
+julia -e 'using Pkg; Pkg.add("LanguageServer")'
+```
+This installs LanguageServer.jl into your global julia environment where the default launch
+command will look for the package.
+
+It is possible to use a separate package environment for the LanguageServer.jl installation.
+For example, to install in a folder `~/.config/nvim/lsp-julia` use the following command:
```

Comment:
Better to use `~/.local/share/nvim/lspconfig`, I would say.

Reply:
Is there not a common convention in Julia for storing user julia environments (eg ~/.virtualenvs for python)?

Reply:
There is `.julia/environments/*` where I guess we could put it. Something like `.julia/environments/nvim-lspconfig/Project.toml` then I guess. I don't have a preference

Reply:
I think that makes sense to me. I would like it to be obvious to users where to wipe the environment, so keeping it with other julia environments is probably easiest

Reply:
Sure, will update.